seacrowd/sea_datasets/phost/phost.py

```diff
@@ -0,0 +1,210 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+from zipfile import ZipFile
+
+import datasets
+import yaml
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{PhoST,
+    title     = {{A High-Quality and Large-Scale Dataset for English-Vietnamese Speech Translation}},
+    author    = {Linh The Nguyen and Nguyen Luong Tran and Long Doan and Manh Luong and Dat Quoc Nguyen},
+    booktitle = {Proceedings of the 23rd Annual Conference of the International Speech Communication Association (INTERSPEECH)},
+    year      = {2022}
+}
+"""
+
+_DATASETNAME = "phost"
+
+_DESCRIPTION = """\
+	PhoST is a high-quality and large-scale benchmark dataset for English-Vietnamese speech translation
+    with 508 audio hours, consisting of 331K triplets of (sentence-lengthed audio, English source
+    transcript sentence, Vietnamese target subtitle sentence).
+"""
+
+_HOMEPAGE = "https://github.com/VinAIResearch/PhoST"
+
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value
+
+_LOCAL = True
+
+_SUPPORTED_TASKS = [Tasks.SPEECH_RECOGNITION, Tasks.SPEECH_TO_TEXT_TRANSLATION, Tasks.MACHINE_TRANSLATION]  
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGUAGES = ["en", "vi"]
```

Comment:
sorry, i'm forgot to mention that we follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
it should be eng and vie

```suggestion
_LANGUAGES = ["eng", "vie"]
```

Reply:
then I need a dict for it. ok then. please wait

Reply:
thanks. done :)